<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgWidget{
background-color: rgb(61, 56, 70);}</string>
   </property>
   <widget class="QLabel" name="dialogName">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Noto Mono&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Edit secret</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>800</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>800</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>680</y>
      <width>340</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(222, 221, 218);
font: 14pt &quot;Noto Mono&quot;;
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>680</y>
      <width>340</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(87, 227, 137);
font: 14pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>800</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="showPasswordButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>310</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
     </font>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generatePasswordButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>310</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
     </font>
    </property>
    <property name="text">
     <string>gen</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="notesEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>800</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="validationError">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(237, 51, 59);
font: 9pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>loginEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>notesEdit</tabstop>
  <tabstop>showPasswordButton</tabstop>
  <tabstop>generatePasswordButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>confirmButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
